web_poet/pages.py

```diff
@@ -100,15 +100,23 @@ def __init_subclass__(cls, skip_nonitem_fields=_NOT_SET, **kwargs):
             return
         cls._skip_nonitem_fields = skip_nonitem_fields
 
-    @validates_input
+    def _validate_input(self) -> None:
```

Comment:
It seems it'd be a bit cleaner if FieldsMixin can handle a case when _validate_input is not defined

Reply:
Yeah, and it's both FieldsMixin and @field that need to handle it, I'll do that and see how it looks.